pkg/database/postgres/deploy.go

```diff
@@ -153,20 +214,65 @@ func DeployPostgresTable(uri string, tableName string, postgresTableSchema *sche
 		}
 
 		statement = RemoveForeignKeyStatement(tableName, currentForeignKey)
-		alterAndDropStatements = append(alterAndDropStatements, statement)
+		foreignKeyStatements = append(foreignKeyStatements, statement)
 
 	NextCurrentFK:
 	}
 
-	for _, alterOrDropStatement := range alterAndDropStatements {
-		if alterOrDropStatement == "" {
-			continue
+	return foreignKeyStatements, nil
+}
+
+func buildIndexStatements(p *PostgresConnection, tableName string, postgresTableSchema *schemasv1alpha2.SQLTableSchema) ([]string, error) {
+	indexStatements := []string{}
+	droppedIndexes := []string{}
+	currentIndexes, err := p.ListTableIndexes("", tableName)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, index := range postgresTableSchema.Indexes {
+		var statement string
+		var matchedIndex *types.Index
+		for _, currentIndex := range currentIndexes {
+			if currentIndex.Equals(types.SchemaIndexToIndex(index)) {
+				goto Next
+			}
+
+			matchedIndex = currentIndex
```

Comment:
this is never hit because `Equals` implementation just returns `false`... At least the one in this PR does.